json/latest/brace-data-exchange.schema.json

```diff
@@ -773,6 +773,37 @@
           "title": "BRP Is Complete",
           "description": "The Borrower Response Package for the loan has been deemed complete by the servicer. If not using the brace servicer platform, this will always be false."
         },
+        "caivrs_eligible":
+        {
+          "$ref": "#/definitions/tribool",
+          "title": "CAIVRS Eligibility",
+          "description": "One or more borrowers appear on the Credit Alert Verification Reporting System."
+        },
+        "has_been_borrower_on_loan_with_prior_fha_claim_within_3yrs": {
+          "$ref": "#/definitions/tribool",
+          "title": "Has Borrower Been Borrower On Loan With Prior FHA Claim Within 3 Years",
+          "description": "One or more borrowers, exception through inheritance, or co-signers on the subject property have had a prior loan on which a FHA partial claim has been paid within the previous three years from evaluation."
```

Comment:
This is probably a product thing but perhaps this copy might read a bit better as something like

> One or more borrowers or co-signers on the subject property, except through inheritance, have had a prior loan on which a FHA partial claim has been paid within the previous three years.